src/bootstrap/dist.rs

```diff
@@ -1274,6 +1272,11 @@ impl Step for Clippy {
         let image = tmp.join("clippy-image");
         drop(fs::remove_dir_all(&image));
         builder.create_dir(&image);
+        
+        if !builder.config.toolstate.miri.testing() {
+            println!("skipping Dist Miri stage{} ({})", stage, target);
+            return None
+        }
```

Comment:
Any reason you do this after the tempdir is already created?

Reply:
Also, uh, this is in the `Clippy` impl...? Shouldn't the same be added for Miri then?

Reply:
I was also about to ask what this `testing` method tests, but CI already says it does not even exist. ;)

`== TestPass` is anyway clearer, IMO.

Reply:
yea... I just did this via copy paste in the github interface. Working with a local build now to get this actually working